projects/PdfDing/demo/module.nix

```diff
@@ -0,0 +1,8 @@
+{ pkgs, ... }:
+{
+  services.pdfding = {
+    enable = true;
+    openFirewall = true;
+    secretKeyFile = pkgs.writeText "django_secret" "foobarbaz";
```

Comment:
We should add a warning that this isn't secure and should not be used in production.